sklearn/utils/tests/test_validation.py

```diff
@@ -687,6 +701,7 @@ def test_suppress_validation():
     assert_raises(ValueError, assert_all_finite, X)
 
 
+@pytest.mark.filterwarnings("ignore: 'warn_on_dtype' is deprecated")  # 0.23
```

Comment:
Instead of ignoring the warning, we should update the test to stop passing the `warn_on_dtype` argument as `test_check_array_series` is actually not concerned with the `warn_on_dtype` thing.

Reply:
done